framework/include/cppmicroservices/Bundle.h

```diff
@@ -398,6 +425,9 @@ class US_Framework_EXPORT Bundle
    * @return The value of the requested property, or an empty string
    *         if the property is undefined.
    *
+   * @deprecated Since 3.0, use GetHeaders() or BundleContext::GetProperty
```

Comment:
We are okay with compiler warnings. Our build system provides a mechanism to temporarily disable treating compiler warnings as errors. There are processes in place to ensure the compiler warnings are addressed eventually.